packages/server/src/middlewares/validate.ts

```diff
@@ -1,12 +1,27 @@
 import { Context } from "koa";
 import { accountSchema } from "../validation";
-export const validateCreateAccountBody = (ctx: Context, next: () => void) => {
+import { getAccountByEmail } from "../db/accounts";
+export const validateCreateAccountData = async (
+  ctx: Context,
+  next: () => void,
+) => {
   const { error } = accountSchema.validate(ctx.request.body);
 
   if (error) {
     ctx.status = 403;
-    ctx.body = "Missing parameters";
+    ctx.message = "Missing parameters";
```

Comment:
It's more a bad request than a 403 Forbidden don't you think ? 
I would have put 
```typescript
   ctx.status = 400;
   ctx.message = "Missing parameters";
```